string name
float64 position
float64 velocity
